recipes/libtiledb-sql-py/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "libtiledb-sql-py" %}
+{% set version = "2.0.1" %}
+{% set sha256 = "e4855cd2a14bd5394ed95e21822c5dccca275ac08b70f7ec875ab2491a825cd6" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ version }}.tar.gz
+  url: https://github.com/TileDB-Inc/TileDB-SQL-Py/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win]
+  skip: True  # [py2k]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+  host:
+    - python
+    - wheel
+    - setuptools
+    - setuptools_scm
+    - setuptools_scm_git_archive
+    - libtiledb-sql
+  run:
+    - python
+    - setuptools
```

Comment:
Do you really need setuptools at runtime?

Reply:
No, this was a mistake, the recipe has been updated.